src/main/java/org/onebusaway/alexa/lib/ObaUserClient.java

```diff
@@ -90,6 +90,21 @@ public ObaUserClient(String obaBaseUrl) throws URISyntaxException {
     }
 
     /**
+     * Returns details about a particular stop, given it's stopId
+     * @param stopId
+     * @return details about a particular stop, given it's stopId
+     */
+    public Optional<ObaStopResponse> getStopDetails(String stopId) {
+        ObaStopResponse response = new ObaStopRequest.Builder(stopId).build().call();
+        log.debug("ObaStopRequest returned " + response.toString());
+        if (response.getCode() == ObaApi.OBA_OK) {
```

Comment:
Ok - just force pushed a new commit that throws an exception if response isn't `OBA_OK`.  I'll update the other REST API calls with a similar format in a separate PR.